Create new WIP channel and fix changelog

MarcelMue opened the pull request:
I messed up the channels stuff because I wasn't thinking. PTAL if this makes sense here.

MarcelMue committed:
Bump version for 1.14.3

MarcelMue committed:
Create new WIP channel and fix changelog

MarcelMue committed:
merge master

MarcelMue commented:
@rossf7 I would like your opinion here - we talked in slack briefly.

rossf7 reviewed (approved):
I'm a bit 😕. So the upgrade was done in `0-7-stable` and this is updating the changelogs retrospectively?

If so I think that's the right approach. If not then I'm missing something!

rossf7 commented:
Ah damn I only meant to comment. If my understanding is correct then I'm fine with merging.

The channels stuff is messy and things will improve when this moves to release CRs.

MarcelMue commented:
So I upgraded on accident here where I did my changes in 0-7-0:
https://github.com/giantswarm/kubernetes-cluster-autoscaler/pull/24/files
Because I was not thinking - this was the mistake I did. 

I now want to make this ready for release so I fixed the changelog and added a new WIP channel. Now I am not sure if this is the way forward.

rossf7 commented:
Yes, the release channels approach is messy but also flexible. I see it as a failed experiment. 

When you make a change you need to decide on the semver. If you just bump the patch version then chart-operator will reconcile clusters on that channel.

If you bump major or minor then you also need to update the channel in cluster-operator. If you want to link cluster-autoscaler to the cluster-operator release then this is the right approach.

MarcelMue commented:
Okay - I will provide the correct PR to cluster-operator and go from there. Thanks for your input :heart: 

Note: This is not really related to any issue with channels but mainly caused by me not being careful / context switching too much and therefor messing up. Just looking to get my poop fixed :heart:

kopiczko commented on CHANGELOG.md:
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v0.8.0] WIP
```
We also need links at the bottom.

MarcelMue replied:
Good call - I added links and fixed the previous versions.
I linked to PRs because the repo does not have folders or github releases.

MarcelMue committed:
Add links to changelog

MarcelMue commented:
Going with this.
